src/Header/SetCookie.php

```diff
@@ -378,7 +378,7 @@ public function getMaxAge()
      */
     public function setExpires($expires)
     {
-        if ($expires === null) {
+        if ($expires === null || $expires === '') {
```

Comment:
@Martin-P that thread is about Expires **HEADER**

> If set to 0, or omitted, the cookie will expire at the end of the session (when the browser closes).
http://php.net/manual/en/function.setcookie.php